data/transport missions.txt

```diff
@@ -1867,6 +1872,297 @@ mission "Terraforming 5"
+	destination "Earth"
+	passengers 1
+	to offer
+		has "Terraforming 9: done"
+	
+	on offer
+		conversation
+			`Amy, Nolan, and the other two scientists tell you the known history of Tundra's climate. Millions of years ago, an asteroid impact or large volcanic eruption altered the planet's atmosphere. Now the planet no longer retains enough heat from its host star in order to stay warm.`
+			`	"Tundra's snow covered surface only exacerbates the problem, reflecting most of the star's light back into space." Nolan explains more. "This means that we need to both alter the atmosphere in order for it retain more heat and alter the surface in order for it to reflect less heat."
+			`	Amy rolls out a map of the entire planet onto the table. Different regions of the surface are color coded, and a large red spot in the planet's southern hemisphere sticks out to you.`
+			`	"Tundra has very low volcanic activity, but seismic mapping has lead us to this area." Amy circles the red spot with her finger. "Magma from the planet's mantle is building up under this area of the crust, which lucky for us is considerably thin. Still, it could be another 10,000 years before the magma finally erupts onto the surface in a supereruption, melting the snow around it and releasing carbon dioxide into the atmosphere. Tundra's crust also has an unusually high amount of serpentinite, which could release methane during the eruption and increase the amount of the star's energy that gets trapped in the atmosphere."`
+			choice
+				`	"So are we going to wait 10,000 years for it to erupt?"`
+				`	"How does this help our situation?"`
+					goto situation
+			
+			`	The Republic representative scoffs at your comment. "If we were then we wouldn't be here, now would we, Captain?"`
+			
+			label situation
+			`	One of the other scientists responds. "What we plan to do is encourage the eruption of this volcano."`
+			`	"How exactly do you plan to do that?" the Tundra representative responds. "And what will happen to the people who live near that area?"`
+			`	Nolan answers the native's questions. "The people who live near the volcano will need to evacuate the area with all their belongings. The Republic should provide any assistance needed for the evacuation. As for how we will cause the volcano to erupt, we need something energetic enough to crack the planet's crust in the location of the volcano, releasing the pressure that the magma is causing to spark an eruption. That's where we bring in another asteroi-"`
+			`	"No, no, no!" the Republic representative exclaims, cutting Nolan off mid sentence. "The Republic will not gain a reputation for crashing asteroids into its own planets. The consequences of such an action could be disastrous should they go wrong! If Tundra wants to terraform their planet that bad then normal techniques should be employed, none of this 'equilibrium mapping' nonsense."`
```

Comment:
`mid sentence` felt weird to me, but I don't know. I'll just point to https://english.stackexchange.com/a/379312.

`that bad` -> `that badly,`
(I recognize "that bad" as an adverbial expression, but it just sounds off in this context.)